R/importNGWMN_wml2.R

```diff
@@ -43,7 +44,9 @@ importNGWMN <- function(input, asDateTime=FALSE, tz="UTC"){
     raw <- TRUE
   } else {
     returnedDoc <- getWebServiceData(input, encoding='gzip')
-    
+    if(is.null(returnedDoc)){
```

Comment:
Since the function doesn't stop in the `getWebServiceData`, we now have to add these all over the place

Reply:
The other way to handle would be a trycatch with a message and return NULL in the catch.